qemu/tests/cfg/single_driver_install.cfg

```diff
@@ -21,6 +24,8 @@
     del devcon_dirname
     vio_driver_chk_cmd = 'driverquery /si | find /i "%s"'
     key_to_install_driver = "tab;kp_enter"
+    need_uninstall = yes
+    need_destroy = yes
```

Comment:
Same with `need_uninstall`,  since all cases use the same rule,  I think adding one step in the script is ok.

Reply:
Actually another case need call this case as well and have the different test logic , so we need keep it.

Reply:
No use for `need_destroy` in `win_virtio_driver_update_test`.   Any other case i missed ?